config/environment.js

```diff
@@ -7,8 +7,8 @@ module.exports = function(environment) {
     baseURL: '/',
     locationType: 'auto',
 
-    authBaseUri: "http://localhost:4000",
-    apiBaseUri: "http://localhost:4001",
```

Comment:
Would like to leave these as the localhost variants, unless there is a reason to change them.